README.md

```diff
@@ -25,3 +25,37 @@ and solve it as we did in the lectures. Give the final $\Theta$ complexity.
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+
+Runtime Analysis of the Divide and Conquer Sort Algorithm:
+
+The final runtime complexity of the implemented divide and conquer sorting algorithm is:
+T(n) = 3T(n/3) + (n)
+T(n) = 3(3T(n/9) + (n/3)) + (n)
+T(n) = 3^2(T)(n/9) + (n) + (n)
+T(n) = 3^3(T)(n/27) + (n) + (n) + (n)
+T(n) = 3^3(T)(n/27) + 3n
+
+T(n) = 3^k(T)(n/3^k) + kn (pattern)
+n/3^k = 1 (stopping condition)
+n = 3^k
+log3(n) = k
+
+T(n) = 3^log3(n) T(1) + n * log3(n)
+T(n) = n * T(1) + n * log3(n)
+T(n) = n(c) + n * log3(n) (T(1) = c, where c is a constant)
+T(n) = O(n log n)
```

Comment:
Notation is still incorrect.

Reply:
Could you elaborate about the notation please? Also for plagiarism note, do you mean the sources? Because if so I will definitely include it, I apologize for forgetting to include it so far as I was focusing on getting this right.

Reply:
You must include the plagiarism note in the syllabus and in the first set of slides. Please use the notation we used throughout in class; I specifically mentioned this multiple times in class throughout the semester.